### DISCUSSION TITLE
Is the project built with rsbuild supported

### DISCUSSION THREAD

**Discussion by JoeXin** [unplugin/unplugin-vue-components#865] [Q&A]
### Clear and concise description of the problem

I hope supported it

### Suggested solution

I hope supported it

### Validations

- [X] Follow our [Code of Conduct](https://github.com/unplugin/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by sxzz** [comment]
You can use it in Rspack/Rsbuild with:

```ts
import Components from 'unplugin-vue-components'

const Plugin = Components.rspack
```

If you encounter any issues, please provide feedback here.

**Comment by 553444MabelMacaulay** [comment]
When troubleshooting, I found here. Currently, the following configuration is OK:
```ts
import { defineConfig } from '@rsbuild/core';
import { pluginVue } from '@rsbuild/plugin-vue';
import { ElementPlusResolver } from 'unplugin-vue-components/resolvers';
import Components from 'unplugin-vue-components/rspack';
import AutoImport from 'unplugin-auto-import/rspack';

export default defineConfig({
  plugins: [pluginVue()],
  tools: {
    rspack: {
      plugins: [
        AutoImport({
          resolvers: [ElementPlusResolver({})]
        }),
        Components({
          resolvers: [ElementPlusResolver({})]
        })
      ]
    }
  }
});
```
